marlowe-runtime/indexer-test/Spec.hs

```diff
@@ -0,0 +1 @@
+{-# OPTIONS_GHC -F -pgmF hspec-discover #-}
```

Comment:
Is this robust against non-nix builds? I'm wondering if the hspec preprocessor will be on the path if someone is doing a straight `cabal build` outside of nix?

Reply:
Yes, it is - it's a GHC plugin loaded via cabal.